Include/pymath.h

```diff
@@ -227,4 +227,10 @@ PyAPI_FUNC(void) _Py_set_387controlword(unsigned short);
  * behavior. */
 #define _Py_InIntegralTypeRange(type, v) (_Py_IntegralTypeMin(type) <= v && v <= _Py_IntegralTypeMax(type))
 
+/* Return the smallest integer k such that n < 2**k, or 0 if n == 0.
+ * Equivalent to floor(lg(x))+1.  Also equivalent to: bitwidth_of_type -
+ * count_leading_zero_bits(x)
```

Comment:
You can please elaborate what is "lg"? Is it the base 2 logarithm function? Usually, it's spelling log2 like math.log2().

Reply:
I prefer the `log2` spelling, too; I think it's less likely to confuse readers. Having said that, the existing `factorial` implementation comments use `lg`.